### PR TITLE
Iot commands

### PR DESCRIPTION
**Pull request by gguuss**
Adds commands example / test to Cloud IoT Sample.

### CODE REVIEW

File: iot/iot.rb [hunk 1]
```diff
@@ -626,6 +626,38 @@
...
+  iot_client = Cloudiot::CloudIotService.new
+  iot_client.authorization = Google::Auth.get_application_default(
+    "https://www.googleapis.com/auth/cloud-platform"
+  )
+
+  # The resource name of the location associated with the project
+  parent   = "projects/#{project_id}/locations/#{location_id}"
+  resource = "#{parent}/registries/#{registry_id}/devices/#{device_id}"
+
+  commandReq = Cloudiot::SendCommandToDeviceRequest.new
```
Comment (frankyn) [inline iot/iot.rb]: s/commandReq/command_request/

File: iot/iot.rb [hunk 2]
```diff
@@ -626,6 +626,38 @@
...
+  commandReq.binary_data = data
+
+  # Set configuration for the provided device
+  res = iot_client.send_project_location_registry_group_device_command_to_device resource, commandReq
```
Comment (frankyn) [inline iot/iot.rb]: s/res/response/g

Comment (gguuss) [reply iot/iot.rb]: The response is actually unused, removing completely, renaming elsewhere.

File: iot/iot.rb (hunk 1 above)
Comment (gguuss) [reply iot/iot.rb]: Done.

**Pull request merged**